docs/reference/01-introduction.md

```diff
@@ -0,0 +1,71 @@
+# Introduction
+
+The Prisma Go client aims to be fully typesafe wherever possible, even for complex queries.
+That's why it uses a functional syntax to make sure every argument you provide matches the type in the database.
+If you change something in the database, and the change is incompatible with the code, you will get a compile time error.
+
+The examples use the following prisma schema:
+
+```prisma
+model Post {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    published Boolean
+    title     String
+    content   String?
+
+    comments Comment[]
+}
+
+model Comment {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    content   String
+
+    post   Post @relation(fields: [postID], references: [id])
+    postID String
+}
+```
+
+## The syntax concept
+
+The generated go client uses a specific and consistent functional API syntax.
+
+A simple line to fetch all posts could look as follows:
```

Comment:
For writing technical content, the css-tricks.com creator mentioned his writing "tone" on a podcast that stuck with me:

> Write like we're going on an adventure together.

Applying this to the above...

"A simple line to fetch all posts could look as follows:"

to

"To fetch all posts, you can do:"